ReviewSite/search-panel/search-panel.css

```diff
@@ -0,0 +1,13 @@
+.search-panel {
+    border: 1px solid black;
+    width: 110px;
+    padding: 20px;
+    margin: 10px;
+    text-align: center;
+    float: left;
+}
+@media (max-width: 500px) {
+    .search-panel {
+        float: unset;
+    }
+}
```

Comment:
<img width="1440" alt="Screenshot 2022-03-14 at 11 44 07" src="https://user-images.githubusercontent.com/43992068/158156784-add3e55a-caca-4763-936f-199c42c274c7.png">
It doesn't look like ur mockup(